CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## master / unreleased
 
+* [CHANGE] Ruler: Better handling of special characters in filenames when mapping them onto the filesystem. #2506
```

Comment:
Sorry to bother you on this, but I think that reading the changelog entry it's not clear why it's a CHANGE. Could you clarify it?

Reply:
I need to give it more thought. I'm starting to think a changelog entry is not needed or if anything it should be labeled a BUGFIX since it aligns Cortex with expected behavior.